src/policy/fees/block_policy_estimator.cpp

```diff
@@ -723,6 +724,29 @@ CFeeRate CBlockPolicyEstimator::estimateFee(int confTarget) const
     return estimateRawFee(confTarget, DOUBLE_SUCCESS_PCT, FeeEstimateHorizon::MED_HALFLIFE);
 }
 
+ForecastResult CBlockPolicyEstimator::EstimateFee(ConfirmationTarget& target)
+{
+    ForecastResult::ForecastResponse response;
+    response.forecaster = ForecastType::BLOCK_POLICY;
+    if (target.type != ConfirmationTargetType::BLOCKS) {
+        return ForecastResult(response, "Incorrect Confirmation target, expecting blocks");
+    }
+    FeeCalculation feeCalcConservative;
+    CFeeRate feerate_conservative{estimateSmartFee(target.value, &feeCalcConservative, /**conservative**/ true)};
+    FeeCalculation feeCalcEconomical;
+    CFeeRate feerate_economical{estimateSmartFee(target.value, &feeCalcEconomical, /**conservative**/ false)};
+    response.current_block_height = feeCalcEconomical.bestheight;
+    if (feerate_conservative == CFeeRate(0) || feerate_economical == CFeeRate(0)) {
+        return ForecastResult(response, "Insufficient data or no feerate found");
+    }
+    // Note: size can be any positive non-zero integer; the evaluated fee/size will result in the same fee rate,
+    // and we only care that the fee rate remains consistent.
+    int32_t size = 1000;
+    response.low_priority = FeeFrac(feerate_economical.GetFee(size), size);
+    response.high_priority = FeeFrac(feerate_conservative.GetFee(size), size);
```

Comment:
this might just be a personal opinion, but "low priority" and "high priority" usually mean different block targets to me. people who are accustomed to the mempool.space wording might say the same. but really, here and in the `MempoolForecaster`, it's a measure of how conservative/confident we are in our estimate no?

![image](https://github.com/user-attachments/assets/11477535-87d2-4ad7-b9ab-4f3d6b95c86a)

Reply:
Good point Yes, it will be better to maintain the conservative and economical name?
Happy to update.

Reply:
This is fixed now.